interface-spec.html

```diff
@@ -106,6 +106,10 @@ <h2>Design elements and principles</h2>
     </li>
     <li>Should allow "upgrading" a plain object into a fully functional triple</li>
     <li>Interfaces may have additional implementation specific properties.</li>
+    <li>
+       We don't define any validation of given values (e.g. IRI, URI, CURIE). 
+       If an implementation enforces valid values, it should state details prominent for the user.
```

Comment:
@k00ni I'm not sure to understand this part:

> it should state details prominent for the user

Could you please explain and we'll find a better wording?

Reply:
Thanks, I get the idea now!

```suggestion
       Implementations that apply validation should make this fact clear in their documentation.
```

Something like this?

Reply:
:+1: